### ISSUE TITLE
Week 17 - 2020-07-12

### ISSUE THREAD

**Issue by JinhangZhu** [JinhangZhu/project-diary#13]
## 计划

- [x] 对训练集运用k-means生成anchors

- [x] 训练YOLOv3模型

- [x] 设计算法接着识别左右手

- [ ] SSD review

- [ ] Batch normalization

**Comment by JinhangZhu** [comment]
### 20200706

对训练集运用`sklearn.cluster.KMeans`计算出来anchors。

```
37, 53
42, 77
45, 90
54, 96
60, 113
67, 135
76, 136
91, 166
100, 203

```

![](https://github.com/JinhangZhu/project-diary/blob/master/images/anchor-kmeans.png?raw=true)

**Comment by JinhangZhu** [comment]
### 20200707

![image](https://user-images.githubusercontent.com/45664104/86856802-fed15780-c0b4-11ea-85f3-c391bd47c5be.png)

### 20200708
在Colab上训练了hand64小数据集，效果非常之差，心情很down，也没想明白什么原因，明天调试一下。
![results](https://user-images.githubusercontent.com/45664104/86968130-8f667100-c163-11ea-88d7-f84634e965ba.png)

### 20200709
尝试了：
- 检查数据集标注
- 检查cfg文件
- 提高学习速率

均无改进。

明天尝试以下。为什么？1）找到好的代码版本，一个exploratory process。2）探寻数据集大小的影响。之前的 #2 只用COCO16结果为什么好？思考一下。

- [x] Reproduce #8 的左右手训练

- [x] 使用全数据集ego-hand训练
